CHANGELOG.md

```diff
@@ -0,0 +1,274 @@
+
+## Other Notable Changes
+
+- Handled the case of invalid action in scheduler module ([#563](https://github.com/kubeedge/kubeedge/pull/563), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- Added nil pointer handling in watcher,go ([#561](https://github.com/kubeedge/kubeedge/pull/561), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- Corrected the topic name being used in bluetooth_mapper ([#559](https://github.com/kubeedge/kubeedge/pull/559), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- Mismatch in access mode names ([#558](https://github.com/kubeedge/kubeedge/pull/558), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- Fix edge_core crashs when SecurityContext.Privileged pointer ([#555](https://github.com/kubeedge/kubeedge/pull/555), [@shouhong](https://github.com/shouhong))
+
+- Supports automatic deletion of edge containers if their pods ([#537](https://github.com/kubeedge/kubeedge/pull/537), [@shouhong](https://github.com/shouhong))
+
+- fix nil pointer panic possibility in deviceController ([#501](https://github.com/kubeedge/kubeedge/pull/501), [@sids-b](https://github.com/sids-b))
+
+- Make file changes to fix #issue 496 ([#497](https://github.com/kubeedge/kubeedge/pull/497), [@srivatsav123](https://github.com/srivatsav123))
+
+- Fixed restart issue of mapper ([#492](https://github.com/kubeedge/kubeedge/pull/492), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- fix EdgeController config keys ([#490](https://github.com/kubeedge/kubeedge/pull/490), [@sids-b](https://github.com/sids-b))
+
+- Added README.MD for Kubeedge installer #issue-477. ([#489](https://github.com/kubeedge/kubeedge/pull/489), [@srivatsav123](https://github.com/srivatsav123))
+
+- fix the broken link ([#481](https://github.com/kubeedge/kubeedge/pull/481), [@lou-lan](https://github.com/lou-lan))
+
+- Device CRD: remove configmap when no device is bound with ([#479](https://github.com/kubeedge/kubeedge/pull/479), [@chendave](https://github.com/chendave))
+
+- Changed exit status from 0 to 1 when bluetooth mapper exits ([#475](https://github.com/kubeedge/kubeedge/pull/475), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- Added Device CRD sample for CC2650 Bluetooth device ([#472](https://github.com/kubeedge/kubeedge/pull/472), [@sujithsimon22](https://github.com/sujithsimon22))
+
+- update setup steps in docs.kubeedge.io ([#470](https://github.com/kubeedge/kubeedge/pull/470), [@edisonxiang](https://github.com/edisonxiang))
+
+- Distinguish debugging information between controller and ([#462](https://github.com/kubeedge/kubeedge/pull/462), [@kadisi](https://github.com/kadisi))
+
+- delete useless field messageLayer ([#459](https://github.com/kubeedge/kubeedge/pull/459), [@kadisi](https://github.com/kadisi))
+
+- Added lableselectors logic to deploy kubeedge edgenode pod to ([#453](https://github.com/kubeedge/kubeedge/pull/453), [@pavan187](https://github.com/pavan187))
```

Comment:
lableselectors -> label selectors

Reply:
https://github.com/kubeedge/kubeedge/pull/453 , this line is same as the PR heading